### DISCUSSION TITLE
Newly created RabbitMQ user cannot connect to an exchange without configure permission

### DISCUSSION THREAD

**Discussion by myasul** [rabbitmq/rabbitmq-server#7960] [Questions] [answered]
I am creating a user and set its permission (`read` & `write`) using the RabbitMQ Management API. But whenever I try to connect to the broker without configure permission via `amqplib` it throws an error `Operation failed: ExchangeDeclare; 403 (ACCESS-REFUSED) with message "ACCESS_REFUSED - access to exchange 'amq.topic' in vhost '/' refused for user`.

When I set the `configure` permission it connects properly. Do the users really need to have `configure` permission just to connect to the broker? I only want them to publish and receive messages.

I set them via using this request:
```
PUT http://localhost:15672/api/permissions/%2F/<username>
{
    configure: '^$',
    write: '^(amq\.topic)$',
    read: '^(amq\.topic)$'
}
```

**Comment by myasul** [comment]
Please disregard this, I fixed the issue by using only `connect` only rather than with `connection.createChannel`.

**Comment by michaelklishin** [comment, marked as answer]
@myasul this is by design: users do not have any permissions unless you grant them some.

Connecting with an AMQP 0-9-1 client without opening a channel is hardly useful.
